lighthouse-core/fraggle-rock/gather/session.js

```diff
@@ -24,22 +23,6 @@ class ProtocolSession {
     this._nextProtocolTimeout = undefined;
     /** @type {WeakMap<any, any>} */
     this._callbackMap = new WeakMap();
-
-    // FIXME: Monkeypatch puppeteer to be able to listen to *all* protocol events.
-    // This patched method will now emit a copy of every event on `*`.
-    const originalEmit = session.emit;
-    // @ts-expect-error - Test for the monkeypatch.
-    if (originalEmit[SessionEmitMonkeypatch]) return;
-    session.emit = (method, ...args) => {
-      // OOPIF sessions need to emit their sessionId so downstream processors can recognize
-      // the target the event came from.
-      const sessionId = this._targetInfo && this._targetInfo.type === 'iframe' ?
-        this._targetInfo.targetId : undefined;
-      originalEmit.call(session, '*', {method, params: args[0], sessionId});
```

Comment:
Ah shoot, kinda overlooked this bit `{method, params: args[0], sessionId}` ... the normal events from puppeteer don't have any of that stuff...

Reply:
fixed